### PR TITLE
add command

### PR DESCRIPTION
**Pull request by davgur**

### CODE REVIEW

File: importer/cusource/remove_files_by_sha1.go [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+		} else {
+			log.Info("Send Rollback of transaction ")
+			err = tx.Rollback()
+			if err != nil {
+				log.Infof("Commit error %s", err)
+			}
+		}
+	}()
+
+	if err != nil {
```
Comment (edoshor) [inline importer/cusource/remove_files_by_sha1.go:79]: error handling should be immediate after error happened. This should be before the `defer`

File: importer/cusource/remove_files_by_sha1.go [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+		log.Infof("Problem on create transaction: %s ", err)
+		return err
+	}
+
+	log.Infof("Start remove files_operations with files ids:%v ", fId)
+	qfo := fmt.Sprintf("DELETE FROM files_operations fo where  fo.file_id = %d", fId)
+	rfo, err := queries.Raw(mdb, qfo).Query()
+	if err != nil {
+		log.Infof("Problem on delete files_operations: %s ", err)
```
Comment (edoshor) [inline importer/cusource/remove_files_by_sha1.go:88]: Errors should be logged with `error` level.

File: importer/cusource/remove_files_by_sha1.go [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+		return err
+	}
+	defer rfo.Close()
+
+	log.Infof("Start remove operations with ids: %v ", oIds)
+
+	if len(oIds) > 0 {
+		qo := fmt.Sprintf("DELETE FROM operations o where  o.id IN (%s)", strings.Join(oIds, ","))
+		ro, err := queries.Raw(mdb, qo).Query()
```
Comment (edoshor) [inline importer/cusource/remove_files_by_sha1.go:97]: do we really need `Raw` here ?
I'd expect something like `models.Operations.Where.DeleteAll`

Comment (edoshor) [inline importer/cusource/remove_files_by_sha1.go:97]: `Query()` is wrong here. It should be `Exec()`

File: importer/cusource/remove_files_by_sha1.go [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+		if err != nil {
+			log.Infof("Problem on delete operations: %s ", err)
+			return err
+		}
+		defer ro.Close()
```
Comment (edoshor) [inline importer/cusource/remove_files_by_sha1.go:102]: not needed after you use `Exec()` above

File: importer/cusource/remove_files_by_sha1.go [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+	qfs := fmt.Sprintf("DELETE FROM files_storages fs where  fs.file_id = %d", fId)
+	rfs, err := queries.Raw(mdb, qfs).Query()
+	if err != nil {
+		log.Infof("Problem on delete files_storages: %s ", err)
+		return err
+	}
+	defer rfs.Close()
+
+	qf := fmt.Sprintf("DELETE FROM files f where  f.id = %d", fId)
+	rf, err := queries.Raw(mdb, qf).Query()
```
Comment (edoshor) [inline importer/cusource/remove_files_by_sha1.go:114]: file.DELETE() instead of Raw

File: importer/cusource/remove_files_by_sha1.go [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+}
+func deleteFileOnTransaction(mdb *sql.DB, fId int64, oIds []string) error {
+	log.Info("Open transaction")
+	tx, err := mdb.Begin()
+	defer func() {
+		if err == nil {
+			log.Info("Send commit of transaction ")
+			err = tx.Commit()
+			if err != nil {
+				log.Infof("Commit error %s", err)
```
Comment (edoshor) [inline importer/cusource/remove_files_by_sha1.go:68]: you swallow error here where it should be returned. No ?

**Review by edoshor — CHANGES_REQUESTED** [review]

File: importer/cusource/remove_files_by_sha1.go (hunk 6 above)
Comment (davgur) [reply importer/cusource/remove_files_by_sha1.go:68]: @edoshor hm, it's defer go func. Why it's need to return anything?

**Pull request closed without merging**